tests/utils/test_databricks_utils.py

```diff
@@ -105,10 +105,10 @@ def test_get_workspace_info_from_databricks_secrets():
 
 def test_get_workspace_info_from_dbutils():
     mock_dbutils = mock.MagicMock()
-    mock_dbutils.notebook.entry_point.getDbutils.return_value.notebook.return_value.getContext.return_value.browserHostName.return_value.get.return_value = (  # noqa
+    mock_dbutils.notebook.entry_point.getDbutils.return_value.notebook.return_value.getContext.return_value.browserHostName.return_value.get.return_value = (  # pylint: disable=line-too-long
```

Comment:
It looks to me that this kind of "line too long" error is something we do want to fix. Can we just group the function call chain in multiple lines?
Same suggestions for the rest of this file.

Reply:
We can break this into multiple lines:

```python
    # fmt: off
    mock_dbutils.notebook.entry_point \
        .getDbutils.return_value \
        .notebook.return_value \
        .getContext.return_value \
        .browserHostName.return_value \
        .get.return_value = "mlflow.databricks.com"
    # fmt: on
```